examples/reminders/lib/feature_reminder_list/views/reminder_list_page.dart

```diff
@@ -39,17 +39,7 @@ class ReminderListPage extends StatelessWidget implements AutoRouteWrapper {
         ),
       );
 
-  AppBar _buildAppBar(BuildContext context) => AppBar(
-        actions: [
-          IconButton(
-            icon: const Icon(Icons.refresh),
-            onPressed: () => context
-                .read<ReminderListBlocType>()
-                .events
-                .loadPage(reset: true),
-          ),
-        ],
-      );
+  AppBar _buildAppBar(BuildContext context) => AppBar();
```

Comment:
Same comment as above

Reply:
I have removed the method and moved the `AppBar()` in the `Scaffold`